samples/unit-tests/series-column/columnrange-datalabels/demo.js

```diff
@@ -0,0 +1,66 @@
+QUnit.test("Change of label alignment after add(#4605)", function (assert) {
```

Comment:
The `columnrange` series type is different from `column`, so this test should be added to the existing `unit-tests/series-columnrange/datalabels/demo.js`.